presentation/feedprofile/src/main/java/com/hilingual/presentation/feedprofile/profile/navigation/FeedProfileNavigation.kt

```diff
@@ -0,0 +1,85 @@
+package com.hilingual.presentation.feedprofile.profile.navigation
+
+import androidx.compose.animation.AnimatedContentTransitionScope
+import androidx.compose.animation.EnterTransition
+import androidx.compose.animation.ExitTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavOptions
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.navigation
+import androidx.navigation.toRoute
+import com.hilingual.core.navigation.Route
+import com.hilingual.presentation.feedprofile.follow.FollowListRoute
+import com.hilingual.presentation.feedprofile.profile.FeedProfileRoute
+import kotlinx.serialization.Serializable
+
+private const val ANIMATION_DURATION = 300
+
+@Serializable
+internal data class FeedProfileGraph(val userId: Long) : Route
```

Comment:
p1
저희 정해진 컨벤션에 맞게 Navigation 그래프의 라우트는 타입 빼주시면 좋을 거 같아용
```suggestion
internal data class FeedProfileGraph(val userId: Long)
```